src/p5.dance.js

```diff
@@ -1499,6 +1648,14 @@ module.exports = class DanceParty {
     console.warn(message);
   }
 
+  setGeneratedDancerMove(source) {
```

Comment:
nit: in a few places we call `this.generatedDancer.setSource` directly but we also have this function. Consolidate in favor of one or the other?